azure/posix.yml

```diff
@@ -80,4 +80,4 @@ jobs:
           artifactFeeds: "sklearn-nightly"
         displayName: Authenticate twine with artifact feed
       - script: |
-          twine upload -r sklearn-wheels-testing/_packaging/sklearn-nightly/pypi/simple/ dist/
+          twine upload -r https://pkgs.dev.azure.com/thomasjpfan/sklearn-wheels-testing/_packaging/sklearn-nightly/pypi/upload dist/
```

Comment:
Alternatively it might be possible to use a `$HOME/.pypirc` file:

```
[distutils]
Index-servers =
  sklearn-nightly

[sklearn-nightly]
Repository = https://pkgs.dev.azure.com/thomasjpfan/sklearn-wheels-testing/_packaging/sklearn-nightly/pypi/upload
```